src/main/kotlin/blackjack/ui/InputView.kt

```diff
@@ -15,9 +13,9 @@ class InputView {
 
     fun askPlayersWantToDrawCard(): Boolean {
         val playerAnswer = readln()
-        if (playerAnswer == "Y") {
+        if (playerAnswer == "Y" || playerAnswer == "y") {
             return true
         }
         return false
```

Comment:
소문자 y도 추가해주셨네요. 👍 
이렇게 대문자 소문자 둘 다 나열하실수도 있겠지만, lowercase를 사용하시면 하나의 조건문으로 해결하실 수 있어요!
```suggestion
        return playerAnswer.lowercase() == "y"
```